templates/terraform/environments/prod/user_access.tf

```diff
@@ -0,0 +1,101 @@
+# define AWS policy documents for developer
+data "aws_iam_policy_document" "developer_access" {
+  # EKS
+  statement {
+    effect    = "Allow"
+    actions   = ["eks:ListClusters"]
+    resources = ["*"]
+  }
+  statement {
+    effect    = "Allow"
+    actions   = ["eks:DescribeCluster"]
+    resources = ["arn:aws:eks:<% index .Params `region` %>:<% index .Params `accountId` %>:cluster/<% .Name %>-prod*"]
+  }
+
+  # ECR
+  statement {
+    effect = "Allow"
+    actions = [
+      "ecr:DescribeImages",
+      "ecr:DescribeRepositories"
+    ]
+    resources = ["*"]
+  }
+
+  # S3
+  statement {
+    effect    = "Allow"
+    actions   = ["s3:ListBucket"]
+    resources = ["arn:aws:s3:::*<% index .Params `productionHostRoot` %>"]
```

Comment:
Hmm if we're going to be using these values a lot more at the root how about putting them into locals in `main.tf`?